agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -96,10 +110,19 @@ function create_sysdig_serviceaccount {
 }
 
 function install_k8s_agent {
-    echo "* Creating sysdig-agent clusterrole and binding"
-    kubectl apply -f /tmp/sysdig-agent-clusterrole.yaml
-    fail=0
-    outbinding=$(kubectl create clusterrolebinding sysdig-agent --clusterrole=sysdig-agent --serviceaccount=$NAMESPACE:sysdig-agent --namespace=$NAMESPACE 2>&1) || { fail=1 && echo "kubectl create clusterrolebinding failed!"; }
+    if [ $OPENSHIFT -eq 0 ]; then
+        echo "* Creating sysdig-agent clusterrole and binding"
+        kubectl apply -f /tmp/sysdig-agent-clusterrole.yaml
+        fail=0
+        outbinding=$(kubectl create clusterrolebinding sysdig-agent --clusterrole=sysdig-agent --serviceaccount=$NAMESPACE:sysdig-agent --namespace=$NAMESPACE 2>&1) || { fail=1 && echo "kubectl create clusterrolebinding failed!"; }
+    else
+        echo "* Creating sysdig-agent access policies"
+        fail=0
+        outbinding=$(oc adm policy add-scc-to-user privileged -n ibm-observe -z sysdig-agent 2>&1) || { fail=1 && echo "oc adm policy add-scc-to-user failed!"; }
```

Comment:
should be `-n $NAMESPACE` instead of `-n ibm-observe`

Reply:
addressed this issue